integration-tests/src/tests/network/peer_manager.rs

```diff
@@ -1,156 +0,0 @@
-use actix::Actor;
-use actix::{Handler, SyncContext};
-use std::sync::Arc;
-
-use near_network::types::{NetworkClientMessages, NetworkClientResponses};
-use near_network_primitives::types::{NetworkViewClientMessages, NetworkViewClientResponses};
-
-#[derive(Default)]
-struct MockClientActor {
-    on_view_client_callback:
-        Option<Arc<dyn Fn(&NetworkViewClientMessages) -> NetworkViewClientResponses + Send + Sync>>,
-}
-
-impl Actor for MockClientActor {
-    type Context = SyncContext<Self>;
-}
-
-impl Handler<NetworkViewClientMessages> for MockClientActor {
-    type Result = NetworkViewClientResponses;
-
-    fn handle(&mut self, msg: NetworkViewClientMessages, _ctx: &mut Self::Context) -> Self::Result {
-        match &self.on_view_client_callback {
-            Some(callback) => callback.as_ref()(&msg),
-            None => match msg {
-                NetworkViewClientMessages::GetChainInfo => NetworkViewClientResponses::ChainInfo {
-                    genesis_id: Default::default(),
-                    height: 1,
-                    tracked_shards: vec![],
-                    archival: false,
-                },
-                _ => NetworkViewClientResponses::NoResponse,
-            },
-        }
-    }
-}
-
-impl Handler<NetworkClientMessages> for MockClientActor {
-    type Result = NetworkClientResponses;
-    fn handle(&mut self, _msg: NetworkClientMessages, _ctx: &mut Self::Context) -> Self::Result {
-        return NetworkClientResponses::NoResponse;
-    }
-}
-
-// This test is trying to see how PeerManager reacts to repeated AnnounceAccounts requests.
-// We want to make sure that we broadcast them only once.
-#[cfg(feature = "test_features")]
-#[test]
-fn repeated_announce_accounts() {
```

Comment:
Do we completely kill this test and `infinite_loop`?

Reply:
This test in an extended version has been moved to chain/network/peer_manager. I've talked with @mfornet about what infinite_loop is actually testing. He says it is about testing TTL, although I fail to see how. Anyway I've added a test for TTL and removed this one.

Reply:
Rechecking the test [`infinite_loop`](https://github.com/near/nearcore/blob/493aabf81155111c66ac6886e505e356f69998bd/integration-tests/src/tests/network/infinite_loop.rs#L90) closely, I can't completely tell what could go wrong. But it was making sure that upon requests of type `SyncRoutingTable` only new accounts were announced, and we were not repeating same accounts.

Not saying that we need to keep the tests though, I know the syncing algorithm have evolved significantly from the time that test was created.